src/utils/markdown/MarkdownRenderer/media.tsx

```diff
@@ -48,5 +49,17 @@ export const getMedia = ({ serverPath }: useMarkdownRendererProps) => {
         />
       );
     },
+    iframe: (props: React.IframeHTMLAttributes<HTMLIFrameElement>) => {
+      const { src, ...rest } = props;
+      return (
+        <iframe
+          width={EMBED_SIZE.w}
+          height={EMBED_SIZE.h}
+          loading="lazy"
+          {...rest}
```

Comment:
Smart to add `{...rest}` here. I was just about to request we add ability to overwrite these until I noticed